README.md

```diff
@@ -1,107 +1,154 @@
 
-- [Copylefted libre software](https://github.com/iv-org/invidious) (AGPLv3+ licensed)
-- Lightweight (the homepage is ~4 KB compressed)
-- No ads
-- No tracking
-- Javascript is 100% optional
-- Tools for managing subscriptions:
-  - Only show unseen videos
-  - Only show latest (or latest unseen) video from each channel
-  - Delivers notifications from all subscribed channels
-  - Automatically redirect homepage to feed
-  - Import subscriptions from YouTube
-- Audio-only mode (and no need to keep window open on mobile)
-- Dark mode
-- Embed support
-- Set default player options (speed, quality, autoplay, loop)
-- Support for Reddit comments in place of YouTube comments
-- Import/Export subscriptions, watch history, preferences
-- [Developer API](https://docs.invidious.io/API.md)
-- Does not use any of the official YouTube APIs
-- No need to create a Google account to save subscriptions
-- No Code of Conduct
-- No Contributor license Agreement
-- Available in many languages, thanks to [Weblate](https://hosted.weblate.org/projects/invidious/)
 
----
+## Documentation
 
-## Screenshots:
+The full documentation can be accessed online at https://docs.invidious.io/
 
-| Player                                                                                                                  | Preferences                                                                                                             | Subscriptions                                                                                                               |
-| ----------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| [<img src="screenshots/01_player.png?raw=true" height="140" width="280">](screenshots/01_player.png?raw=true)           | [<img src="screenshots/02_preferences.png?raw=true" height="140" width="280">](screenshots/02_preferences.png?raw=true) | [<img src="screenshots/03_subscriptions.png?raw=true" height="140" width="280">](screenshots/03_subscriptions.png?raw=true) |
-| [<img src="screenshots/04_description.png?raw=true" height="140" width="280">](screenshots/04_description.png?raw=true) | [<img src="screenshots/05_preferences.png?raw=true" height="140" width="280">](screenshots/05_preferences.png?raw=true) | [<img src="screenshots/06_subscriptions.png?raw=true" height="140" width="280">](screenshots/06_subscriptions.png?raw=true) |
+The documentation's source code is available in this repository:
+https://github.com/iv-org/documentation
 
----
+### Extensions
 
-## Donate:
+We highly recommend the use of [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect), a
```

Comment:
Add a link to https://addons.mozilla.org/firefox/addon/privacy-redirect/ and https://chrome.google.com/webstore/detail/privacy-redirect/pmcmeagblkinmogikoikkdjiligflglb (or https://github.com/SimonBrazell/privacy-redirect#get )

Reply:
Their project's README already have big buttons for that, so is it really necessary?

Reply:
It's a bit more user-friendly yes.